### ISSUE TITLE
Users of the FirePad demo should be warned that the pad expires after some hours

### ISSUE THREAD

**Issue by xylo** [firebase/firepad#261]
Users of the FirePad demo should be warned that the pad expires after some hours and that the document gets deleted.

**Comment by mikelehen** [comment]
Can you provide more details?  That definitely shouldn't be the case.

**Comment by xylo** [comment]
Of course.  Two days ago I created the pad https://demo.firepad.io/#20AVfAvEEA.  I edited the pad together with some other people, but when I opened the pad yesterday our edited content was gone and the welcome text appeared again:

> Welcome to your own private pad!
> 
> Share the URL below and collaborate with your friends.

**Comment by mikelehen** [comment]
Ah!  Yes, I didn't actually realize it but it looks like the pads get wiped nightly.  We should indeed update the text to make that more clear.  Sorry for the inconvenience!

**Comment by pgnn** [comment]
Hello,
when I am on the main page (https://firepad.io/#1) and create a private pad. How many people can work simultaneously (limit)? Just want to make sure that via the button "private pad" I don't create a demo pad with limits I don't know. Thank you

**Comment by mikelehen** [comment]
No specific limit (20 people should be fine).  But as noted above, those pads are meant for demo purposes only and get deleted every day, so it is not a very reliable tool for important notes (sorry). :-)

**Comment by pgnn** [comment]
Thanks for the answer. The demo gets deleted everyday. How is the deletion connected to day time and location. Is it like: deletion every day, New York, midnight? Or is it: Demo created -> 24 hours later it'll be deleted.

**Comment by mikelehen** [comment]
Same time every day.  I don't actually know what time it happens.  Sorry.

**Comment by katowulf** [comment]
Just to point out the obvious, this is an OSS project and the demo is a _demo_, not an app for you to use in production. Rather than trying to work around the constraints of the demo, one should use the app as intended.

In other words, if you want a permanent, reliable version for your personal or business use, you can [follow the download instructions](https://github.com/firebase/firepad#downloading-firepad), point the build to your own Firebase database, utilize our free hosting to deploy your app, and enjoy reliability and uptime unprecedented by trying to rely on a demo!

**Comment by xylo** [comment]
Some more tips for setting up your own firepad:
* Firebase setup:
  * go to https://firebase.google.com/
  * create a Firebase project and open it
  * go to "database" and then to "rules"
  * paste the following content in the "rules" text field:
```
{
  "rules":{
    "$secretid":{
      "history": {
        ".read": true,
        "$revision": {
          /* Prevent overwriting existing revisions. */
          ".write": "data.val() === null"
        }
      },
      "checkpoint": {
        ".read": true,
        /* Ensure author of checkpoint is the same as the author of the revision they're checkpointing. */
        ".write": "root.child($secretid).child('history').child(newData.child('id').val()).child('a').val() === newData.child('a').val()",
        ".validate": "newData.hasChildren(['a', 'o', 'id'])"
      },
      "users": {
        ".read": true,
        "$user": {
          ".write": true
        }
      }
    }
  }
}
```
* then go to the project settings in the firebase console
  * here you'll find the "Project ID" which is used as "<DATABASE_NAME>" in the example and your API key.  Fill both in and it should work.
